intrinsic_funcs/08_vector_reduction_functions.md

```diff
@@ -271,270 +271,270 @@ vuint64m1_t vredxor_vs_u64m2_u64m1 (vuint64m1_t dest, vuint64m2_t vector, vuint6
-vint64m1_t vredxor_vs_i64m8_i64m1_m (vbool8_t mask, vint64m1_t dest, vint64m8_t vector, vint64m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8mf8_u8m1_m (vbool64_t mask, vuint8m1_t dest, vuint8mf8_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8mf4_u8m1_m (vbool32_t mask, vuint8m1_t dest, vuint8mf4_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8mf2_u8m1_m (vbool16_t mask, vuint8m1_t dest, vuint8mf2_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8m1_u8m1_m (vbool8_t mask, vuint8m1_t dest, vuint8m1_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8m2_u8m1_m (vbool4_t mask, vuint8m1_t dest, vuint8m2_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8m4_u8m1_m (vbool2_t mask, vuint8m1_t dest, vuint8m4_t vector, vuint8m1_t scalar, size_t vl);
-vuint8m1_t vredxor_vs_u8m8_u8m1_m (vbool1_t mask, vuint8m1_t dest, vuint8m8_t vector, vuint8m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16mf4_u16m1_m (vbool64_t mask, vuint16m1_t dest, vuint16mf4_t vector, vuint16m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16mf2_u16m1_m (vbool32_t mask, vuint16m1_t dest, vuint16mf2_t vector, vuint16m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16m1_u16m1_m (vbool16_t mask, vuint16m1_t dest, vuint16m1_t vector, vuint16m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16m2_u16m1_m (vbool8_t mask, vuint16m1_t dest, vuint16m2_t vector, vuint16m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16m4_u16m1_m (vbool4_t mask, vuint16m1_t dest, vuint16m4_t vector, vuint16m1_t scalar, size_t vl);
-vuint16m1_t vredxor_vs_u16m8_u16m1_m (vbool2_t mask, vuint16m1_t dest, vuint16m8_t vector, vuint16m1_t scalar, size_t vl);
-vuint32m1_t vredxor_vs_u32mf2_u32m1_m (vbool64_t mask, vuint32m1_t dest, vuint32mf2_t vector, vuint32m1_t scalar, size_t vl);
-vuint32m1_t vredxor_vs_u32m1_u32m1_m (vbool32_t mask, vuint32m1_t dest, vuint32m1_t vector, vuint32m1_t scalar, size_t vl);
-vuint32m1_t vredxor_vs_u32m2_u32m1_m (vbool16_t mask, vuint32m1_t dest, vuint32m2_t vector, vuint32m1_t scalar, size_t vl);
-vuint32m1_t vredxor_vs_u32m4_u32m1_m (vbool8_t mask, vuint32m1_t dest, vuint32m4_t vector, vuint32m1_t scalar, size_t vl);
-vuint32m1_t vredxor_vs_u32m8_u32m1_m (vbool4_t mask, vuint32m1_t dest, vuint32m8_t vector, vuint32m1_t scalar, size_t vl);
-vuint64m1_t vredxor_vs_u64m1_u64m1_m (vbool64_t mask, vuint64m1_t dest, vuint64m1_t vector, vuint64m1_t scalar, size_t vl);
-vuint64m1_t vredxor_vs_u64m2_u64m1_m (vbool32_t mask, vuint64m1_t dest, vuint64m2_t vector, vuint64m1_t scalar, size_t vl);
-vuint64m1_t vredxor_vs_u64m4_u64m1_m (vbool16_t mask, vuint64m1_t dest, vuint64m4_t vector, vuint64m1_t scalar, size_t vl);
-vuint64m1_t vredxor_vs_u64m8_u64m1_m (vbool8_t mask, vuint64m1_t dest, vuint64m8_t vector, vuint64m1_t scalar, size_t vl);
+vint8m1_t vredsum_vs_i8mf8_i8m1_m (vbool64_t mask, vint8m1_t dest, vint8mf8_t vector, vint8m1_t scalar, size_t vl, uint8_t ta);
```

Comment:
I think reductions can use dest==vundefined for ta. The mask doesn't affect which elements are written in the destination.